README.md

```diff
@@ -207,19 +207,20 @@ _To get a local copy up and running follow these simple example steps._
 2. **Docker**: We use Docker Desktop, but the Docker CLI will also work.
 3. **Create a Copy of .npmrc**: In the project directory root, duplicate `.npmrc.example`, saving the copy as `.npmrc`.
 
-    >   ```bash
-    >   # root project directory
-    >   cp .npmrc.example .npmrc
-    >   ```
+   > ```bash
+   > # root project directory
+   > cp .npmrc.example .npmrc
+   > ```
 
 4. **Create and Add Your Npm and GitHub Tokens**: In order to connect with the Dataverse API, developers will need to install [@iqss/dataverse-client-javascript][dv_repo_dvclientjs_npm_url] from the GitHub registry by following the steps outlined below. Read more about access tokens on [GitHub Docs][dv_docs_github_userauthtoken_url].
 
-    >  **Getting a GitHub Token**
-    >  1. Go to your GitHub [Personal Access Tokens][dv_docs_github_token_url] settings
-    >  2. Select `Generate new token (classic)`
-    >  3. Give the token a name and select scope `read:packages`
-    >  4. Copy the generated token and replace the string `YOUR_GITHUB_AUTH_TOKEN` in the previously created `.npmrc` file.
-    >       Now, you should be able to install the [Dataverse JavaScript][dv_repo_dvclientjs_url] client using npm.
+   > **Getting a GitHub Token**
+   >
+   > 1. Go to your GitHub [Personal Access Tokens][dv_docs_github_token_url] settings
+   > 2. Select `Generate new token (classic)`
+   > 3. Give the token a name and select scope `read:packages`
+   > 4. Copy the generated token and replace the string `YOUR_GITHUB_AUTH_TOKEN` in the previously created `.npmrc` file.
+   >    Now, you should be able to install the [Dataverse JavaScript][dv_repo_dvclientjs_url] client using npm.
 
 Afterwards, your .npmrc file should resemble the following:
```

Comment:
Should we add a note? saying that the .npmrc file is still not identical as .npmrc.example, but because we will cover that later in the readme ( I think you already cover that in the design system deployment readme right?) 
Or it will be too mucht text?